src/component/Legend.tsx

```diff
@@ -67,16 +84,8 @@ function getDefaultPosition(style: React.CSSProperties, props: Props, box: Bound
 
 export type Props = DefaultProps & {
   wrapperStyle?: CSSProperties;
-  chartWidth?: number;
-  chartHeight?: number;
   width?: number;
   height?: number;
-  margin?: {
```

Comment:
Did this represent chart margin or specific legend margin?

Reply:
generateCategoricalChart passes in the chart margin, and ignores whatever users put in to Legend props.

Reply:
Cool thanks